### ISSUE TITLE
V4 Development: DefineConstants requires an overhaul

### ISSUE THREAD

**Issue by danielmarbach** [aws/aws-sdk-net#3326]
### Describe the feature

The constants defined in the project have grown over the years and are a bit messy. There is still BCL constants defined and some of the constants look like they are the standard constants but are actually not. It would be good to do a general sweep over the v4 development branch and get rid of unnecessary constants, BCL definitions and rely as much as possible on the constants already defined by the SDK.

Also some of the defined constants are unnecessarily specific. For example it would be enough with the current target frameworks to use NETSTANDARD, NETCOREAPP and NET and once NETCOREAPP is dropped just NETSTANDARD and NET.

### Use Case

Less cognitive overhead for everyone involved

### Acknowledgements

- [ ] I may be able to implement this feature request
- [ ] This feature might incur a breaking change

### AWS .NET SDK and/or Package version used

All

### Targeted .NET Platform

All

### Operating System and version

All

**Comment by normj** [comment]
Agreed they do need to be clean up. These constants started getting used in the SDK during the pre 1.0 of .NET Core and we have never gone back to clean them up to use the standard constants. I'll try and take a look how hard it would be to switch to the standard constants.

**Comment by normj** [comment]
Closing as we part of V4 and the upcoming V4.1 we have for the most part switched to the standard `DefineContants`.
